src/iosource/Packet.h

```diff
@@ -208,19 +213,20 @@ class Packet {
 	 */
 	bool l3_checksummed;
 
-	/**
-	 * Indicates whether the packet should be processed by zeek's
-	 * session analysis in NetSessions.
-	 */
-	bool session_analysis;
-
 	/**
 	 * Indicates whether this packet should be recorded.
 	 */
 	mutable bool dump_packet;
 
-	// Wrapper to generate a packet-level weird. Has to be public for packet analyzers to use it.
-	void Weird(const char* name);
+	// These are fields passed between various packet analyzers. They're best
```

Comment:
Would be good to document the semantics of these fields: what do they capture, and when can they be expected to be set.

Reply:
Added comments for each field.